internal/storage/object.go

```diff
@@ -31,4 +31,5 @@ type MinObject struct {
 	MetaGeneration int64
 	Updated        time.Time
 	Metadata       map[string]string
+	IsGzipEncoded  bool
```

Comment:
I first wondered if `IsGzipEncoded` should be a separate member or an item in `Metadata` dict. But as `Metadata` and `ContentEncoding` are separate in struct `gcs.Object`, it is clear that the same should be done here.

Reply:
Metadata and content-encoding are different propoerties on gcs object definition. lets not club them into one.

https://cloud.google.com/storage/docs/json_api/v1/objects

Reply:
Ok. Keeping them separate as we both agree on it.